ocaml/database/db_cache.ml

```diff
@@ -27,7 +27,7 @@ module Remote_db : DB_ACCESS = Db_rpc_client_v1.Make(struct
     let initialise () =
```

Comment:
I did think about that, but it changes quite a lot more code as the initialise function is part of a module signature that several modules implement. A further tidy-up, perhaps.